go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -0,0 +1,217 @@
+package ovndbmanager
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/klog"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+func RunDBChecker(stopCh <-chan struct{}) {
+	defer utilruntime.HandleCrash()
+	klog.Info("Starting DB Checker to ensure cluster membership and DB consistency")
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		ensureOvnDBState(util.OvnNbdbLocation, stopCh)
+	}()
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		ensureOvnDBState(util.OvnSbdbLocation, stopCh)
+	}()
+	<-stopCh
+	klog.Info("Shutting down db checker")
+	wg.Wait()
+	klog.Info("Shut down db checker")
+}
+
+func ensureOvnDBState(db string, stopCh <-chan struct{}) {
+	ticker := time.NewTicker(60 * time.Second)
+	klog.Infof("Starting ensure routine for Raft db: %s", db)
+	_, _, err := util.RunOVSDBTool("db-is-standalone", db)
```

Comment:
do we want to be more specific with handling the error code here? from the docs
```
Tests whether db is a database file in clustered or standalone format, respectively. If so, exits with status 0; if not, exits with status 2. (Exit status 1 indicates an error, e.g. db is not an OVSDB database or does not exist.)
```

If the code is 1 it indicates an error and we should probably log that and maybe restart the checker?

Reply:
Fixed.